### ISSUE TITLE
GDAL not found in GCBM container

### ISSUE THREAD

**Issue by aornugent** [moja-global/FLINT.Cloud#72]
GDAL 3.4.1 is built in the `flint.base` image along with it's python bindings on [L157](https://github.com/moja-global/FLINT.Cloud/blob/f1063fc717928fbe71b16305291c0d7492deb387/local/rest_api_gcbm/Dockerfile.base#L157), but python doesn't know where to find it.

```python
import gdal 
ModuleNotFoundError: No module named 'gdal'
```

If I append the `osgeo` package (the parent of GDAL) to PYTHONPATH I get a different error:

```
export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3/dist-packages/osgeo
```

```python
import gdal
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3/dist-packages/osgeo/gdal.py", line 2039, in <module>
    from . import ogr
ImportError: attempted relative import with no known parent package
```

**Comment by aornugent** [comment]
False alarm - I can access it from the osgeo package

```python
from osgeo import gdal
```
